### PR TITLE
Move more controllers to a sane directory structure

### PR DESCRIPTION
**Pull request by dkanada**

### CODE REVIEW

File: src/scripts/routes.js [hunk 1]
```diff
@@ -16,52 +16,80 @@ define([
...
     defineRoute({
         path: "/addserver.html",
         autoFocus: false,
         anonymous: true,
         startup: true,
-        controller: "addserver"
+        controller: "auth/addserver"
+    });
+    defineRoute({
+        path: "/mypreferencesdisplay.html",
```
Comment (JustAMan) [inline src/scripts/routes.js]: why `.html` templates were not moved?

File: src/scripts/routes.js [hunk 2]
```diff
@@ -229,57 +252,21 @@ define([
...
-        autoFocus: false,
-        transition: "fade",
-        controller: "user/subtitles"
-    });
     defineRoute({
         path: "/notificationsetting.html",
         autoFocus: false,
         roles: "admin",
-        controller: "notificationsetting"
+        controller: "dashboard/notifications/notification"
```
Comment (JustAMan) [inline src/scripts/routes.js]: wtf, why there are "notification" and "notifications"?..

File: src/scripts/routes.js [hunk 3]
```diff
@@ -391,17 +373,18 @@ define([
         roles: "admin",
         controller: "userprofilespage"
     });
+
     defineRoute({
         path: "/wizardremoteaccess.html",
         autoFocus: false,
         anonymous: true,
-        controller: "wizardremoteaccess"
+        controller: "wizard/access"
```
Comment (JustAMan) [inline src/scripts/routes.js]: I would name the controller `wizard/remoteaccess`, as it's a different thing than just `access` IMHO

File: src/scripts/routes.js [hunk 4]
```diff
@@ -191,19 +219,14 @@ define([
...
     });
     defineRoute({
         path: "/login.html",
         autoFocus: false,
         anonymous: true,
         startup: true,
-        controller: "loginpage"
-    });
-    defineRoute({
-        path: "/metadataadvanced.html",
```
Comment (JustAMan) [inline src/scripts/routes.js]: why was this removed?

File: src/scripts/routes.js [hunk 5]
```diff
@@ -16,52 +16,80 @@ define([
...
 
     defineRoute({
         path: "/addplugin.html",
         autoFocus: false,
         roles: "admin",
-        controller: "addpluginpage"
+        controller: "dashboard/plugins/add"
     });
     defineRoute({
-        path: "/autoorganizelog.html",
```
Comment (JustAMan) [inline src/scripts/routes.js]: why was this removed?

File: src/scripts/routes.js [hunk 6]
```diff
@@ -16,52 +16,80 @@ define([
...
-        roles: "admin"
+        path: "/mypreferencesmenu.html",
+        autoFocus: false,
+        transition: "fade",
+        controller: "user/menu"
     });
     defineRoute({
-        path: "/channelsettings.html",
```
Comment (JustAMan) [inline src/scripts/routes.js]: why was this removed?

File: src/scripts/routes.js [hunk 7]
```diff
@@ -345,11 +332,6 @@ define([
         roles: "admin",
         controller: "streamingsettings"
     });
-    defineRoute({
-        path: "/support.html",
```
Comment (JustAMan) [inline src/scripts/routes.js]: why this was removed?

File: src/videoosd.html [hunk 8]
```diff
@@ -1,27 +1,19 @@
...
+        <div class="osdPoster"></div>
         <div class="osdControls">
-
             <div class="osdTextContainer osdMainTextContainer">
                 <h3 class="osdTitle"></h3>
                 <div class="osdMediaInfo"></div>
                 <div class="osdMediaStatus hide">
-                    <i class="md-icon animate">autorenew</i> <span>${FetchingData}</span>
+                    <i class="md-icon animate">autorenew</i>
+                    <span>${FetchingData}</span>
```
Comment (JustAMan) [inline src/videoosd.html]: is indentation correct here?

File: src/scripts/routes.js (hunk 1 above)
Comment (dkanada) [reply src/scripts/routes.js]: None of the templates have been moved so far, only controllers.

File: src/scripts/routes.js (hunk 2 above)
Comment (dkanada) [reply src/scripts/routes.js]: The latter is the list showing all of them while the former is the settings page for a single notification.

File: src/scripts/routes.js (hunk 4 above)
Comment (dkanada) [reply src/scripts/routes.js]: doesn't exist

File: src/scripts/routes.js (hunk 5 above)
Comment (dkanada) [reply src/scripts/routes.js]: doesn't exist

File: src/scripts/routes.js (hunk 6 above)
Comment (dkanada) [reply src/scripts/routes.js]: doesn't exist afaik

File: src/scripts/routes.js (hunk 7 above)
Comment (dkanada) [reply src/scripts/routes.js]: definitely doesn't exist

File: src/videoosd.html (hunk 8 above)
Comment (dkanada) [reply src/videoosd.html]: It is now, it definitely wasn't before though.

Comment (JustAMan) [reply src/videoosd.html]: Right, I didn't notice closing `</i>`
